Makefile

```diff
@@ -36,6 +36,7 @@ clean:
 check: default linting
 	tests/generate.py
 	tests/cli.py
+	python3-coverage run --append --omit=tests/\* -- tests/internals.py
```

Comment:
Please make tests/internals.py  --> tests/test_replug.py or something similar that is more descriptive of the use of that file. The test_ prefix is important, let's use 'nosetests3 -v --with-coverage' if anything, so that additional tests that do not require running the binaries directly (e.g. real unit tests of code functions) can be detected automatically, that way we won't have to make more changes to Makefile when new unittests are added (see the try-command branch).